### PR TITLE
Add mostly complete compilation

### PR DESCRIPTION
**Pull request by vtrlx**
As stated in the commit: "Compilation is added and works - save for one slight problem. The compiler does not output to the user. Orion has blocked alerts coming from sandboxed plugins, and the other method to show output to the user breaks XMLHttpRequests which compilation relies on."

To see output from the compiler, open a developer console (F12). The user will need to refresh the page to see the resulting files.

Consider this my final pull for now. I'll be mostly unable to contribute until the end of my exam period (April 8th), but after that I'll be able to continue with fixing the bug. In the mean time, I will provide small write-ups on each issue outlining what must be done for those.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,69 @@ This is a plugin to extend the [Orion] text editor to support features allowing
...
+    git clone https://git.eclipse.org/r/orion/org.eclipse.orion.client.git
+
+Copy the Umple.Orion files into the cloned repo
+
+    cp -r umple.orion/org.eclipse.orion.client/* org.eclipse.orion.client/
+
+Clone and build the server repo (note: the Orion Client and Server repositories must be in the same directory for Orion to build.)
+
+    git clone https://git.eclipse.org/r/orion/org.eclipse.orion.server.git
+    mvn clean install -P platform-kepler,local-build -DskipTests
```
Comment (vahdat-ab) [inline README.md:40]: I tried to build, but it looks like it needs a pom.xml file.
screenshot from 2016-03-31 15 01 54

any idea about the error?

**Comment by vtrlx** [comment]
You need to `cd org.eclipse.orion.server`, then run the Maven build. It will automatically build the client as well.

**Comment by vahdat-ab** [comment]
I've tried to run the script in both folders. It didn't work. is there any specific dependency nedded?

**Comment by vahdat-ab** [comment]
I changed my OS and switched to Windows. It's built. I wonder if I need to install clietn plugin as a separate process? Now, it's no highlighting and it doesn't show the compile menu.

**Comment by vahdat-ab** [comment]
Looks like I have to add it manually. I wonder if there is a way to pack it to build process?
I installed and when I see highlights but the menu for compiling doesn't come. Last time, I saw it but it doesn't come now. any guide?

**Comment by vtrlx** [comment]
I've fixed a problem with a path being incorrect. Try building again now.

**Comment by vahdat-ab** [comment]
I tired. It doesn't run umple files that have `use` statement.

**Pull request closed without merging**